imap_processing/tests/codice/test_codice_l1a.py

```diff
@@ -205,7 +205,8 @@ def test_l1a_logical_sources(test_l1a_data, index):
     expected_logical_source = f"imap_codice_l1a_{DESCRIPTORS[index]}"
 
     # Write the dataset to a file to set the logical source attribute
-    _ = write_cdf(processed_dataset)
+    filename = write_cdf(processed_dataset)
+    logger.warning(f"Wrote CDF file to {filename}")
```

Comment:
I dislike logs/prints in tests in general because they spam the output, but I think this should at least be an info level instead of a warning level if you are going to add it.
```suggestion
    logger.info(f"Wrote CDF file to {filename}")
```

Reply:
I will be removing this -- I am just using it to debug some GitHub Actions build errors.

Reply:
@greglucas Speaking of which -- I was hoping to use the L1a CDFs generated by my tests to be used as input files for L1b testing, but I can't figure out where exactly in the GitHub Actions directory structure these L1a files are being written to. And I can't seem to get the `logger.info` or `logger.warning` messages (which I was hoping would tell me where the file is being written to) to even appear in the build output.  Do you have any insight on how I could figure this out?

Reply:
They should be created in temporary directories (We set IMAP_DATA_DIR to a temp path before every single test for isolation). This pattern will only work if you create them within the same test and not across test functions because the cleanup happens at the end of every test to consider them isolated.

Reply:
Thanks, good to know. Sounds like I should go down the 'download from S3' route for these then.